docs/avt1guiance.md

```diff
@@ -0,0 +1,53 @@
+3. With the plugin installed and your token copied head into DAP's options found inside the plugin dropdown menu and paste the token into the Authentication form and hit apply.
+4. Now that DAP's authenticated the Supported Rulesets section in the options menu of the plugin should be populated. Typically you're going to want the latest ruleset unless someone has told you otherwise.
+5. Hit the apply button at the bottom of the page.
+
+## Using DAP👷
+
+Let's DAP! It's pretty straight forward, but there are a few gotchas. For consistency's sake we'll use DAP on the [Dynamic Assessment Plug-in Homepage](https://ibm.biz/BdYkM9).
+
+Head over to the page and hit `cmd-option-c` to open up your dev tools. DAP is tucked away inside a drop down menu inside a panel. From the default Elements panel hit the >> icon below the X on the far right. You should see `DAP`. Click that to fire up DAP for this page.
+
+_Be Warned: You can only run DAP with internal IBM rulesets if you're connected to W3. On an IBM campus this isn't a problem, but if you're abroad fire up the Cisco AnyConnect Secure Mobility Client._
+
+Hit the play button and wait for the scan to finish. **We've got violations!**. At the time of writing there are 7 Violations, 9 Potential Violations, and 2 Manual Checks. There's also multiple "frames", but if you're testing single components or a group of components in a Storybook environment you can ignore those✋.
+
+On the main frame (the one open by default) there's a detailed list of all violation types. Clicking into each violation type gives you more nested dropdowns -- one for each violation found on the page of that specific type. Each violation's dropdown gives you two further options:
+
+1.  A description of the general violation with a ? icon that when clicked will take you to further documentation detailing the violation and most helpfully how to fix it.
+2.  A description of this particular instance of a violation (say for instance an image with no alt attribute). And a crossed out eyeball you can click to ignore this violation which is helpful when printing reports for posterity's sake.
+
+## Running an AVT1 Audit📋
+
+Whenever any new UI code is introduced or existing code is changed or updated we should run DAP on the new code so we can make sure it meets or exceeds IBM's accessibility standards.
+
+DAP is actually mega thorough and can catch a lot of obvious errors in your markup, but it comes with a caveat. **A mountain of false positives**. Typically we're not auditing a webpage or a website as a whole, but a single component in isolation -- sadly this was not DAP's intended purpose and so it throws a lot of false positive violations that you'll first need to sift through and ignore if we're going to be keeping and recording this audit. If you're just testing while you develop or before you push a PR, feel free to just disregard them.
```

Comment:
Can we tighten up the language in:

```md
DAP is actually mega thorough and can catch a lot of obvious errors in your markup, but it comes with a caveat. **A mountain of false positives**.
```

Would be great to use a touch less hyperbole, maybe:

```md
DAP is always evolving its rulesets but unfortunately it can report false positives when testing exclusively on a component.
```